app/components/transaction-item/transaction-item.tsx

```diff
@@ -168,7 +169,7 @@ export const TransactionItem: React.FC<TransactionItemProps> = ({
           >
             {primaryCurrency === "BTC" && tx.settlementCurrency === WalletCurrency.BTC
               ? satAmountDisplay(tx.settlementAmount)
-              : usdAmountDisplay(usdAmount)}
+              : convertUsdToDisplayCurrency(usdAmount)}
```

Comment:
From my understanding, `convertUsdToDisplayCurrency` is not relevant in transation-items or transaction-detail-screens because we will be using what the display currency was at the time as that will be the only thing we have saved with the tx in the ledger. For example if you make some transactions with euros as your display currency and then you switch to another display currency, we will still show euros for that past transaction, because in the ledger we have recorded the euro display amount.  We are not able to convert to the new display currency because we would also need the historical exchange rate for that tx.

Reply:
Instead for any component involving past transactions, they just need to be capable of taking a `PaymentAmount<Curreny>` and be able to properly display that.

Reply:
Yeah agreed but for now this is to test the currency set in debug screen.